vllm/worker/hpu_model_runner.py

```diff
@@ -64,6 +65,26 @@
 LORA_WARMUP_RANK = 8
 
 
+class Singleton(type):
```

Comment:
maybe you could use lru_cache max_count=1 on HPUBucketingGlobalState constructo instead? but this is ok